packages/core-components/api-report.md

```diff
@@ -1447,6 +1454,12 @@ export class UserIdentity implements IdentityApi {
   signOut(): Promise<void>;
 }
 
+// @public
+export const useSidebar: () => SidebarContextType;
+
+// @public
+export const useSidebarPinState: () => SidebarPinStateContextType;
```

Comment:
Feel `SidebarPinStateContextType` -> `SidebarPinState` would be worth it, deprecating the old.